src/shell/Shell.cpp

```diff
@@ -923,6 +928,28 @@ PersistentRefHolder<ContextRef> createEscargotContext(VMInstanceRef* instance, b
     return context;
 }
 
+uint64_t g_lastTick = 0;
+
+static uint64_t fastTickCount()
+{
+#if defined(CLOCK_MONOTONIC_COARSE)
+    timespec ts;
+    clock_gettime(CLOCK_MONOTONIC_COARSE, &ts);
+    return (uint64_t)ts.tv_sec * 1000UL + ts.tv_nsec / 1000000UL;
+#else
+    return tickCount();
+#endif
+}
+
+static void PrintGCMemory(void* data)
```

Comment:
Could you convert first char to lower case?

Reply:
oh my mistake